README.rst

```diff
@@ -49,15 +41,15 @@ Enable tagging on your DX types with a behavior:
   </property>
 
 
-Optionally define tag roots (AT or DX):
+Optionally define tag roots:
 
 ::
 
   <class class="path.to.my.class">
     <implements interface="ftw.tagging.interfaces.tagging.ITagRoot" />
   </class>
 
-For DX, you could alternatively use a behavior to define tag roots:
+You could alternatively use a behavior to define tag roots:
```

Comment:
I think the behavior should be the primary way (or perhaps only way) of specifying a tag root :-)